### ISSUE TITLE
KITTI:Overfitting

### ISSUE THREAD

**Issue by vivi0604** [maudzung/SFA3D#43]
I'm really appreciate you for the nice source code，I download your code and train with the Kitti dataset， the val loss curve is as follows:
![image](https://user-images.githubusercontent.com/38038575/119749356-8ab8c180-bec9-11eb-908c-04324e985ca7.png)
When I use the fpn_resnet_18_epoch_300.pth to pretrain, the val loss curve is as follows:
![image](https://user-images.githubusercontent.com/38038575/119749956-cdc76480-beca-11eb-8f65-533eba88de15.png)

Can you give us some tips to make it work?

**Comment by vivi0604** [comment]
> I'm really appreciate you for the nice source code，I download your code and train with the Kitti dataset， the val loss curve is as follows:
> ![image](https://user-images.githubusercontent.com/38038575/119749356-8ab8c180-bec9-11eb-908c-04324e985ca7.png)
> When I use the fpn_resnet_18_epoch_300.pth to pretrain, the val loss curve is as follows:
> 
![image](https://user-images.githubusercontent.com/38038575/119749926-bd16ee80-beca-11eb-897d-2c833815faf6.png)

> Can you give us some tips to make it work?
